react/features/base/environment/utils.js

```diff
@@ -8,3 +9,28 @@ import Platform from '../react/Platform';
 export function isMobileBrowser() {
     return Platform.OS === 'android' || Platform.OS === 'ios';
 }
+
+/**
+ * Checks whether the chrome extensions defined in the config file are installed or not.
+ *
+ * @returns {Promise[]}
+ */
+export function checkChromeExtensionsInstalled() {
+    const isExtensionInstalled = info => new Promise(resolve => {
+        const img = new Image();
+
+        img.src = `chrome-extension://${info.id}/${info.path}`;
+        img.onload = function() {
+            resolve(true);
+        };
+        img.onerror = function() {
+            resolve(false);
+        };
+    });
+    const extensionInstalledFunction = info => isExtensionInstalled(info);
+    const chromeExtensionBanner = APP.store.getState()['features/base/config'].chromeExtensionBanner || {};
```

Comment:
This function shouldn't access the global APP. Pass the state along please, it will help a lot moving things to react, APP may not live forever.